{{cookiecutter.repo_slug}}/.github/ISSUE_TEMPLATE/01_BUG_REPORT.md

```diff
@@ -18,11 +18,11 @@ assignees: ""
 
 **Expected behavior:**
 
-<!-- Describe what the behavior would be without the bug. -->
+<!-- Describe what you expect the behavior to be without the bug. -->
 
 **Steps to reproduce:**
 
-<!--  Please explain the steps required to duplicate the issue, especially if you are able to provide a sample application. -->
+<!-- Explain the steps required to duplicate the issue, especially if you are able to provide a sample application. -->
```

Comment:
"Please" is used inconsistently in these comments. Since it was on the first one, I left it there, but removed it on subsequent items. (Conceivably you could preface every sentence in here with "Please..." but that seems overkill.)